.circleci/config.yml

```diff
@@ -52,6 +52,11 @@ commands:
       - run:
           name: Deploy application
           command: |
+            if [ << parameters.stage >> = "production" ]; then
+              export NEXT_PUBLIC_GTM_TOKEN_ID=GTM-PWR54HF
```

Comment:
Could do the
```yaml
export NEXT_PUBLIC_GTM_TOKEN_ID=<<parameters.ANALYTICS_PROD_GTM>>
```
and set it within CCI like the rest of the vars so it's not hardcoded.